### ISSUE TITLE
[Raid] Ulduar Alagalon the Observer

### ISSUE THREAD

**Issue by Grindall1** [WoWManiaUK/Redemption#904]
The collapsing Stars in the Alagalon fight evade ranged attacks until hit by a melee.

Collapsing stars spawn throughout the fight, and they have to be killed in order to spawn black holes. Should melee kill the stars, the black holes will port them into another phase, so it has to be done by a ranged.

how stars should work: 
Assign one or two ranged DPS to kill the Collapsing Stars one at a time. Since they slowly lose health on their own (1% health per second), if left alone, they will all explode at the same time and wipe the raid. Alert the healers so they are ready before a Collapsing Star is killed, as the entire raid takes roughly 16,000 shadow damage regardless of positioning (a tank who is too low on health when a star explodes can die and likely cause a wipe). When a star dies, it leaves behind a Black Hole, which is both a void zone and a portal to another realm. Collapsing Stars keep spawning throughout the fight until Phase 2 and should always be killed.
http://wow.gamepedia.com/Algalon_the_Observer_(tactics)

When a ranged attempted to attack a star, it simply evaded. We found that we could solve this issue by having a melee hit the star first, allowing the ranged to then attack it.

**Comment by wcslanker** [comment]
This information is correct. Even being way over geared it is not possible to fight through these bugs.

**Comment by Emcitementz** [comment]
I don't see the problem with this ? Stars don't regen their health when hit by a ranged spell unless it's not what i should look for ?

**Comment by wcslanker** [comment]
No.. what was said earlier, the stars have to be attacked by melee first... otherwise range will get nothing but "evade"

**Comment by Emcitementz** [comment]
Well, i hit them with a ranged spell first and any of the stars were evading.

**Comment by Ford577** [comment]
update to Alagalon:

- it seems you are now able to hit the collapsing stars with ranged attacks without an issue.

- Alagalon periodically resets during the fight. Specifically, if the tank that has aggro move into a black hole during big bang the fight resets. There are other times that the fight resets randomly, and I still don't know the cause. This obviously can cause issues other than annoyance since you only have an hour to kill the boss.

-Constellations. When drug over black holes they are supposed to close them and dissapear, which is they do; however, there are a few issues. First, the use their arcane barrage on whoever has aggro, which is supposed to be random regardless of who has threat. While it may not seem like a big issue since they come from the sky the healers obviously get all aggro to start, and it is possible that there is far too many than there should (see below) but I can't confirm that yet. So unless you have a tank actively collecting all the stars with high resist gear/big healers it is near impossible to do the fight. 

- Black holes. Black spawn after a collapsing stars are killed, which they do; however their animation is bugged. The black hole spawns with a purple portal in it, and when a constellation is drug over it the purple portal goes away but the black hole animation stays on the ground. This causes issues when the raid tries to run through for big bang to a portal that looks like it is there but isn't. 

I recorded a few of our attempts, though it is poor quality, and can work on attempting to upload it should you need to see it.

**Comment by Emcitementz** [comment]
> Constellations. When drug over black holes they are supposed to close them and dissapear, which is they do; however, there are a few issues. First, the use their arcane barrage on whoever has aggro, which is supposed to be random regardless of who has threat. While it may not seem like a big issue since they come from the sky the healers obviously get all aggro to start, and it is possible that there is far too many than there should (see below) but I can't confirm that yet. So unless you have a tank actively collecting all the stars with high resist gear/big healers it is near impossible to do the fight. 

That's not right, "whoever has aggro", the MT will have the highest aggro for sure but he's not being targeted by the constellations.
The selection is purely random but one player will be out of this selection in any case and that's the MT.

**Comment by Ford577** [comment]
Sorry if I might have worded it poorly, let me try again.

https://www.youtube.com/watch?v=fRfVDgTLMGA
@1:43 is when the constellations first show up. 
@ 2:50 is when he explains them.

"They have aggro in a sense they will follow the person with the highest threat, but the person they follow isn't necessarily the person they cast on." 

This ladder part of the description is what I was attempting to refer to. They follow the people with threat and reset every so often like is shown in the video, but they also cast on whoever has threat. What is happening is as they are falling down and are still outside of aggro range for abilities, they are targeting healers due to healer aggro when healers cast heals (like when you pull a boss with a taunt but a healer can take aggro simply by healing since no damage has been done to the boss). This makes too many arcane barrages going off on our healers before we have much of a chance at collecting them. We solved this by placing vigilance on the main tank to allow for insta taunt cooldowns in order to pick up the constellations before they wrecked our healers. Since there is an obsessive amount of constellations that spawn the damage was too much to be handled via healing through (we had three healers as well).

healers getting aggro right away is fine, but only targeting the person who has aggro with the barrage isn't is what I was trying to say. Just wanted to make sure you understood that part in case there was any confusion. I will attempt to upload a video soon to be able to show you what I'm talking about.

**Comment by Emcitementz** [comment]
There was a confusion indeed but it's clear now, thanks.
